tests/AbstractProcessTest.php

```diff
@@ -333,4 +406,16 @@ private function getPhpBinary()
 
         return $runtime->getBinary();
     }
+    
+    private function getPhpCommandLine($phpCode)
+    {
+    	$cmd = $this->getPhpBinary() . ' -r ' . escapeshellarg($phpCode);
+    	
+    	if (defined('PHP_WINDOWS_VERSION_BUILD')) {
+    		// Windows madness! for some obscure reason, the whole command lines needs to be
+    		// wrapped in quotes (?!?)
+    		$cmd = '"'.$cmd.'"';
+    	}
```

Comment:
Is this documented anywhere as a known PHP quirk? I don't recall seeing this done in Symfony's Process component, from which much of the React code was ported.